### DISCUSSION TITLE
闪退，最近特别频繁。只要开始下载多个文件，就会闪退

### DISCUSSION THREAD

**Discussion by fs1314** [leiurayer/downkyi#872] [Q&A]
闪退，最近特别频繁。只要开始下载多个文件，就会闪退

**Comment by 1tobef** [comment]
我更加可悲 🥲🥲🥲 粘贴网址,开始搜索就已经闪退>>>>>>>
1. 我目前在马来西亚, 需要翻墙回国内吗?
2. 我目前没有 +86 电话不能注册用户, 不懂`是否是这原因, 哪位大神可以借个 B 站户口尝试 🫰🫰
3. .net 已经是 4.8, WIN 11, 360 安全卫士, 尝试了 DownKyi-1.5.9 , DownKyi-1.5.8-fix1 , 都是一样 搜索闪退

**Comment by fs1314** [comment]
不清楚，我是win10，win11是垃圾












碧翠子
***@***.***


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月27日(周五) 中午12:26
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [leiurayer/downkyi] 闪退，最近特别频繁。只要开始下载多个文件，就会闪退 (Discussion #872)




 
我更加可悲 🥲🥲🥲 粘贴网址,开始搜索就已经闪退&gt;&gt;&gt;&gt;&gt;&gt;&gt;
  
我目前在马来西亚, 需要翻墙回国内吗?
 
我目前没有 +86 电话不能注册用户, 不懂`是否是这原因, 哪位大神可以借个 B 站户口尝试 🫰🫰
 
.net 已经是 4.8, WIN 11, 360 安全卫士, 尝试了 DownKyi-1.5.9 , DownKyi-1.5.8-fix1 , 都是一样 搜索闪退
